### PR TITLE
i18n(es): Updated `netlify.mdx` & `server-side-rendering.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `netlify.mdx` based on this commit b2390b58726e8928c86d415e29e2883b8d194458 & `server-side-rendering.mdx` based on these commits 278f33ec366aa379e1919be63418d67fa28e48f2, a6ca6f780e996a7812a193174dde1718dd683102 & 8500b624490f88498b7d24a1c45d6350cfd094fd

### CODE REVIEW

File: src/content/docs/es/guides/server-side-rendering.mdx [hunk 1]
```diff
@@ -333,7 +333,7 @@ const factData = await factResponse.json();
 <p>{factData.fact}</p>
 ```
 
-La página Astro a continuación, utilizando estos componentes, puede renderizar partes de la página más rápido. Las etiquetas `<head>`, `<body>` y `<h1>` ya no se bloquean por las solicitudes de datos. El servidor luego obtendrá los datos de `RandomName` y `RandomFact` en paralelo y transmitirá el HTML resultante al navegador.
+La página Astro a continuación, utlizando estos componentes, puede renderizar partes de la página más rápido. Las etiquetas `<head`>, `<body>` y `<h1>` ya no se bloquean por las solicitudes de datos. El servidor luego obtendrá los datos `RandomName` y `RandomFact` en paralelo y transmitirá el HTML resultante al navegador.
```
Comment (Waxer59) [inline src/content/docs/es/guides/server-side-rendering.mdx:336]: Esto creo que esta mal cerrado en el original tambien
```suggestion
La página Astro a continuación, utlizando estos componentes, puede renderizar partes de la página más rápido. Las etiquetas `<head>`, `<body>` y `<h1>` ya no se bloquean por las solicitudes de datos. El servidor luego obtendrá los datos `RandomName` y `RandomFact` en paralelo y transmitirá el HTML resultante al navegador.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo un comentario

**Comment by Waxer59** [comment]
Envie una PR sobre ese typo https://github.com/withastro/docs/pull/3502

**Comment by dreyfus92** [comment]
Gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 💜

**Pull request merged**